ghu_web/ghu_global/static/ghu_global/css/style.css

```diff
@@ -14,6 +14,7 @@ body {
     margin: 0px;
     padding: 4px;
     text-align: right;
+    background-color: #00e673
```

Comment:
It is only for the Login bar.  I just wanted to differentiate it from the rest of the page.